hack/make-rules/verify.sh

```diff
@@ -165,15 +166,16 @@ function run-checks {
       fi
     fi
     echo -e "Verifying ${check_name}"
-    local start=$(date +%s)
+    local start
+    start=$(date +%s)
```

Comment:
```
In ./hack/make-rules/verify.sh line 167:
    local start=$(date +%s)
          ^---^ SC2155: Declare and assign separately to avoid masking return values.
```